system-addon/content-src/styles/activity-stream-windows.scss

```diff
@@ -5,3 +5,21 @@
 .search-wrapper input:focus {
   box-shadow: 0 0 0 1px $blue-50;
```

Comment:
Similarly, can you make this `$os-search-focus-shadow-radius: 1px` and set it to `3px` for the other platforms?